tools/performance/GC.wprp

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WindowsPerformanceRecorder Version="1.0">
+  <Profiles>
+    <EventCollector Id="EventCollector_MyEventSource" Name=".NETSession">
+      <BufferSize Value="1024" />
+      <Buffers Value="1024" />
+    </EventCollector>
+	<EventProvider Id=".NETCommonLanguageRuntimeGCTracking" Name="e13c0d23-ccbc-4e12-931b-d9cc2eee27e4" >
+	 <Keywords>
+	 <!-- GC 0x1 and 0x8000 Exceptions -->
+	 <Keyword Value="8001"/>
+	 </Keywords>
+	</EventProvider>
+    <Profile Id="GC.Verbose.File" Name="GC" Description=".NET GC Tracking"  Base="" LoggingMode="File" DetailLevel="Verbose">
+      <Collectors Operation="Add">
```

Comment:
Without a System collector this profile won't be usable by itself (it won't have process, thread, or loader events) - the user will always have to start at least one other session alongside this in order to make sense of the data in WPA.

Reply:
Yep, I'm removing this profile.